fetcher_push.sh

```
@@ -0,0 +1,13 @@
+./mill service[2.12.18].assembly && \
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)